examples/webgl_loader_sea3d_morph.html

```diff
@@ -187,6 +182,9 @@
 
 				requestAnimationFrame( animate );
 
+				// Update SEA3D Animations
+				THREE.SEA3D.AnimationHandler.update( delta );
```

Comment:
You use multiple mixers?

In case you do, it would probably be better to use a single one and simply go `mixer.animationAction( clip, targetObject ) ... .play()` to start playback. It's more efficient, since it allows for memory reuse.